.changeset/fast-items-film.md

```diff
@@ -0,0 +1,25 @@
+---
+"thirdweb": minor
+---
+
+Adds a defaultSmsCountryCode configuration option to In-App and Ecosystem Wallets
+
+```ts
+createWallet("inApp", {
+    auth: {
+      options: [
+        "google",
+        "discord",
+        "telegram",
+        "email",
+        "passkey",
+        "phone",
+        "farcaster",
+        "line",
+      ],
+      mode: "redirect",
+      passkeyDomain: getDomain(),
+      defaultSmsCountryCode: "AF",
```

Comment:
i would just show this option, maybe with auth option "phone"

kinda confusing to see passkey/mode in here too